test/onnx/test_fx_op_consistency.py

```diff
@@ -661,7 +662,11 @@ def _run_test_output_match(
                     atol = None
                 # Run the test
                 test_suite.run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
-                    model, inputs, rtol=rtol, atol=atol
+                    model,
+                    inputs,
+                    rtol=rtol,
+                    atol=atol,
+                    skip_dynamic_shapes_check=skip_dynamic_shapes_check,
```

Comment:
I don't think we need dynamic_shapes_check in test_fx_op_consistency.py. The goal of this test is to make sure the operator is functional from inputs to outputs in exporter. The focus is on passes and input/attributes processing. Dynamic_shapes_check should be model-wise.